src/ucp/api/ucp.h

```diff
@@ -707,6 +710,32 @@ typedef struct ucp_worker_params {
      * wakeup.
      */
     unsigned                events;
+
+    /**
+     * User data associated with the current worker.
```

Comment:
a pointer to ..

Reply:
this does not have to be a pointer to. User could also place integer value and cast it to a pointer.

Reply:
I would just to be more explicit about it. User data (64 bit object, or pointer)